### ISSUE TITLE
Tracker for things to move here

### ISSUE THREAD

**Issue by domenic** [whatwg/infra#6]
- [x] https://html.spec.whatwg.org/#encoding-terminology
  - code unit, character?, Unicode character?, code-unit length
- https://html.spec.whatwg.org/#case-sensitivity-and-string-comparison
  - case-sensitive comparison, prefix match
  - defaulting of string comparisons to case-sensitive
- [x] https://html.spec.whatwg.org/#common-parser-idioms
  - many things already done but under different names; HTML will need updating
  - White_Space characters, control characters, uppercase/lowercase hex digits
  - Algorithms like "collect a sequence of characters" and friends
  - Update: everything moved except White_Space and control characters
- https://html.spec.whatwg.org/#numbers
  - Maybe only the definitions, not the parsing algorithms? Since the parsing algorithms seem kind of HTML specific?
- https://html.spec.whatwg.org/#dates-and-times ??? wait for a second consumer?
- https://html.spec.whatwg.org/#colours honestly this feels like it should go in some CSS spec?
- [x] https://html.spec.whatwg.org/#space-separated-tokens (concepts left in HTML, parsing moved here)
- [x] https://html.spec.whatwg.org/#comma-separated-tokens (concepts left in HTML, parsing moved here)
- [x] https://dom.spec.whatwg.org/#ordered-sets
  - Has redundancies with a few other things
- [x] https://html.spec.whatwg.org/#namespaces
  - shared between HTML and DOM it seems
- [x] MIME type stuff at https://html.spec.whatwg.org/#resources
  - Moved to MIMESNIFF
- [x] https://html.spec.whatwg.org/#terminology
  - Definition of "or"

**Comment by tobie** [comment]
What about: https://html.spec.whatwg.org/#in-parallel

**Comment by domenic** [comment]
> What about: https://html.spec.whatwg.org/#in-parallel

Hmmm borderline. I think of that as belonging more in HTML since HTML defines the event loop architecture. But I'm not sure.

**Comment by tobie** [comment]
>  I think of that as belonging more in HTML since HTML defines the event loop architecture.

Arguably, there's the "in parallel" abstract concept which belongs in infra, and its event-loop "binding" which belongs in HTML. (But I'm linking to other stuff in the HTML spec anyway, so… I won't argue very strongly.)

**Comment by annevk** [comment]
I think at some point we need to formalize "in parallel" with a concept of threads and message passing. That could maybe live here, but not sure.

**Comment by annevk** [comment]
Thoughts on OP:

* code unit: see #73.
* character, Unicode character: I think these should be dropped. Use code point and scalar value instead. Alternatively we make these acceptable alternate names, but I don't think that'll help clarity or reader comprehension.
* code-unit length: filed #74, happy to fix once we pick between the alternatives.
* case-sensitive comparison, defaulting of string comparisons to case-sensitive: seems reasonable though this gets into the question as to whether we should define "is"/"equals" for a bunch of things
* prefix match: is this used outside AppCache somewhere?
* No opinion on numbers / date and time / color (is to define type=color btw, can't really move to CSS since CSS color concept is much more broad)
* ordered sets: should probably be cleaned up indeed

**Comment by annevk** [comment]
Everything here has now been followed up on or has its own issue.
